### DISCUSSION TITLE
Change billing email address

### DISCUSSION THREAD

**Discussion by ghost** [community/community#102757] [New to GitHub]
Hi there, I want to change the email address that my invoices are sent to. How can I do this? Thanks

**Comment by queenofcorgis** [comment]
Hey there @MindtraceAdmin 👋🏼 

Thanks for coming on down to the community. [This doc ](https://docs.github.com/en/billing/managing-your-github-billing-settings/setting-your-billing-email) should help get you sorted here. Cheers!

**Comment by ghost** [comment]
Thanks for the article. It is odd because the correct email address is showing there ***@***.******@***.***>) but I am getting notifications sent to ***@***.******@***.***>. See attached. Is there somewhere else I need to change this?

Thanks

**Reply by queenofcorgis** [reply]
@MindtraceAdmin I would send a message on our [Support page](https://support.github.com/contact) to see what is going on here!
